tests/processes/ProcessingTest.py

```diff
@@ -234,6 +241,7 @@ def test_dead_processes(self):
             ctrlq, {}, {}, {},
             lambda *args: self.queue.put(args[2]),
             Section(""),
+            None,
```

Comment:
@sils1297 I've added a test that should verify this, I think [[Line]](https://github.com/coala-analyzer/coala/pull/2016/files#diff-f4c53753be8ec5d77fa199b20ac1caa4R115).